app/uk/gov/hmrc/helptosavefrontend/config/FrontendAppConfig.scala

```diff
@@ -72,23 +78,23 @@ object FrontendAppConfig extends AppConfig with ServicesConfig {
     s"Basic: ${new String(base64Encode(s"$user:$password"), encoding)}"
   }
 
-  val nsiCreateAccountUrl: String = s"${baseUrl("nsi")}${getString("microservice.services.nsi.create-account-url")}"
+  val nsiCreateAccountUrl: String = s"${baseUrl("nsi")}/nsihts/createaccount"
 
-  val nsiUpdateEmailUrl: String = s"${baseUrl("nsi")}${getString("microservice.services.nsi.update-email-url")}"
+  val nsiUpdateEmailUrl: String = s"${baseUrl("nsi")}/nsihts/updateemail"
 
-  val nsiHealthCheckUrl: String = s"${baseUrl("nsi")}${getString("microservice.services.nsi.health-check-url")}"
+  val nsiHealthCheckUrl: String = s"${baseUrl("nsi")}/nsihts/healthcheck"
```

Comment:
the health check and update email urls are actually always going to be `nsihts/createaccount` in reality - `nsihts/updateemail` and `nsihts/healthcheck` only exist in our stub , so these need to be read from config